packages/utils/src/time.ts

```diff
@@ -12,3 +12,14 @@ export const durations = {
     return days * durations.hours(24)
   }
 }
+
+/**
+ * Compares timestamps to find out if "value" has expired.
+ * @param value timestamp to compare with
+ * @param now timestamp example: `new Date().getTime()`
+ * @param ttl in milliseconds
+ * @returns true if it's expired
+ */
+export function isExpired(value: number, now: number, ttl: number): boolean {
+  return value + ttl < now
```

Comment:
I'm not sure whether this method really does what it should

Reply:
well, are the tests written for this wrong? can you elaborate?

Reply:
looks good to me. Just not sure whether this needs an extra method. I thought you had something different in mind